src/engraving/dom/symbol.cpp

```diff
@@ -84,6 +96,16 @@ PropertyValue Symbol::getProperty(Pid propertyId) const
     switch (propertyId) {
     case Pid::SYMBOL:
         return PropertyValue::fromValue(m_sym);
+    case Pid::SCORE_FONT:
+        if (m_scoreFont) {
+            return PropertyValue::fromValue(String::fromStdString(m_scoreFont->name()));
+        } else {
+            return PropertyValue::fromValue(nullptr);
```

Comment:
Maybe using an empty string rather than a nullptr looks better (and same below)?